packages/inscription-view/src/components/parts/common/permission/Permission.tsx

```diff
@@ -1,11 +1,11 @@
 import type { StartPermission } from '@axonivy/process-editor-inscription-protocol';
 import { IVY_EXCEPTIONS } from '@axonivy/process-editor-inscription-protocol';
-import RoleSelect from '../responsible/RoleSelect';
+import MultipleRoleSelect from '../role/MultipleRoleSelect';
```

Comment:
`MultpleRoleSelect` is a 1:1 copy of `RoleSelect`. `RoleSelect` is still used for task responsible. The goal is to use als `MultpleRoleSelect` for task responsible, then I can remove `RoleSelect` and rename `MultpleRoleSelect` back to `RoleSelect`